### DISCUSSION TITLE
[list]hide all sample dashboard and chart

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#18444] [Ideas]
As a Superset user, I would like to have the option to hide all sample dashboards from the list/card view. 
currently in Superset there's no content organization ability, also no way to restore deleted objects. Sample dashboards and charts are helpful for first-time users, and can be a good reference from time to time. but they can get really annoying when mixing with users' own content in the list as they grow.....

Proposed change: 1) ability to hide all samples - short term. 2) folder - long term 
<img width="1825" alt="Screen Shot 2021-02-12 at 5 39 38 PM" src="https://user-images.githubusercontent.com/67837651/107838138-4d388a00-6d59-11eb-8e7f-372740de4a9c.png">

**Comment by ferBV** [comment]
I really agree with this petition, that could make superset easier to manage.
